server/opts.go

```diff
@@ -2373,6 +2375,14 @@ func parseJetStream(v any, opts *Options, errors *[]error, warnings *[]error) er
 					return &configErr{tk, fmt.Sprintf("%s %s", strings.ToLower(mk), err)}
 				}
 				opts.JetStreamMaxCatchup = s
+			case "max_buffer_size":
+				s, err := getStorageSize(mv)
+				if err != nil {
+					return &configErr{tk, fmt.Sprintf("%s %s", strings.ToLower(mk), err)}
+				}
+				opts.JetStreamMaxBufferSize = s
+			case "max_buffer_msgs":
+				opts.JetStreamMaxBufferMsgs = int(mv.(int64))
```

Comment:
Will panic is presented as a string "22"

Reply:
I can fix that, but wondering if that's typical?